pkg/sm/sm.go

```diff
@@ -121,25 +121,10 @@ func (sm *ServiceManager) Run() {
 // ServiceManagerBuilder type is an extension point that allows adding additional filters, plugins and
 // controllers before running ServiceManager.
 type ServiceManagerBuilder struct {
+	*web.API
```

Comment:
maybe embed without the star

Reply:
it would require modification down a few levels. Not sure if that's ok. Pointer would be ok I think